lib/cylc/config.py

```diff
@@ -83,16 +83,9 @@ def check_varnames(env):
 
 
 class SuiteConfigError(Exception):
-    """
-    Attributes:
-        message - what the problem is.
-        TODO - element - config element causing the problem
-    """
-    def __init__(self, msg):
-        self.msg = msg
+    """Catch all exception for suite configuration errors."""
+    pass
 
-    def __str__(self):
-        return repr(self.msg)
```

Comment:
This will probably break the test battery, if we are happy with this approach (removing the `repr` logic) I'll go through and fix any failures.

Reply:
It broke everything last time I tried this. Good luck! :wink: